apps/web/src/components/V2/CardV2.tsx

```diff
@@ -1,28 +1,7 @@
 import { classed, deriveClassed } from "~/utils/classed";
 
-type PropsV2 = {
-  onClick?: () => void;
-  children?: React.ReactNode;
-  background?:
-    | "accent-black"
-    | "accent-white"
-    | "grey1"
-    | "grey2"
-    | "grey3"
-    | "grey4"
-    | "accent-yellow-dark"
-    | "accent-yellow-regular"
-    | "accent-blue-dark"
-    | "accent-blue-regular"
-    | "accent-green-dark"
-    | "accent-green-regular"
-    | "red-dark"
-    | "red-regular"
-    | "accent-beige-dark"
-    | "accent-beige-regular";
-};
-const StyledCardV2 = classed.div(
-  "border-2 mx-2 rounded-md shadow-m px-[25px] py-[35px] mb-[20px] ${area}",
+export const CardV2 = classed.div(
```

Comment:
couple notes on migration:
- these belong in `app/components` directory
- don't need to suffix the names of file/component/prop with V2 because they already live in a different folder